docusaurus/docs/publish-a-plugin/publishing-best-practices.md

```diff
@@ -0,0 +1,140 @@
+
+**Key points:**
+
+- **Test compatibility across versions:** Ensure your plugin works seamlessly with various versions of Grafana by setting up E2E tests targeting multiple releases.
+- **Automate testing:** Integrate E2E testing into your continuous integration (CI) pipeline to catch issues early and frequently, reducing potential problems during review.
+
+For a comprehensive guide on setting up E2E tests, see our [E2E test a plugin](../e2e-test-a-plugin/introduction.md) section.
+
+## Validate your plugin
+
+Before submitting your plugin for review, you should validate your plugin to ensure that it meets Grafana's standards for functionality, security, and structure. The easiest way to do this is by using the Plugin Validator tool. This tool checks for potential issues that could prevent your plugin from being accepted, such as security vulnerabilities or structural problems.
+
+**Key points:**
+
+- **Run locally or in CI:** You can run the validator locally or integrate it into your CI workflow to automate the validation process. Note, the validator runs automatically during the default release workflow.
+- **Validation reports:** The tool generates a report, highlighting any errors or warnings that need to be addressed before submission.
+
+For more information on using the validator, refer to the [Plugin Validator documentation](https://github.com/grafana/plugin-validator).
+
+## Provide a provisioned test environment
+
+Provisioning a test environment for your plugin can significantly reduce the review time and make it easier for others to test and contribute to your plugin. A provisioned environment includes a pre-configured Grafana instance with sample dashboards and data sources that demonstrate your plugin's functionality.
+
+**Key points:**
+
+- **Why provisioning matters:** It ensures that both reviewers and contributors can quickly verify your plugin's behaviour without manual setup, speeding up the review and collaboration process.
+- **Automated setup:** You can provision test environments using Docker to create an out-of-the-box experience that replicates a typical Grafana setup.
+
+To learn more about setting up provisioned environments, check out our [provisioning guide](../publish-a-plugin/provide-test-environment.md).
+
+## Automating releases with GitHub Actions
+
+To streamline your plugin development workflow, it's a best practice to automate releases using GitHub Actions. Automating this process helps ensure that your plugin is built, signed, and packaged correctly on each release, reducing human error and expediting the publishing process.
+
+**Key points:**
+
+- **Continuous integration (CI):** Use GitHub Actions to automatically build and test your plugin on every commit or pull request, catching issues early.
```

Comment:
```suggestion
- **Continuous integration:** Use GitHub Actions to automatically build and test your plugin on every commit or pull request, catching issues early.
```

Reply:
this feels strange to me personally, CI is far more commonly referenced than spelling it out